### PR TITLE
fetches datasources and blacklist from the local filesystem initially

### PR DESCRIPTION
**Pull request by p2h5**
Fix for https://github.com/hm-seclab/YAFRA/issues/38

### CODE REVIEW

File: iocextractor/core/server.py [hunk 1]
```diff
@@ -4,7 +4,6 @@
 
 # pylint: disable=C0413, C0411
 
-from logging import debug
```
Comment (DerAlexmeister) [inline iocextractor/core/server.py:7]: Remove this line

File: iocextractor/core/server.py [hunk 2]
```diff
@@ -270,6 +275,7 @@ def __call__(self, app, *args, **kwargs):
         '''
         create_topic_if_not_exists(KAFKA_SERVER, IOC_TOPIC_NAME)
         scheduler.start()
+        Extractor.refetch_blacklist()
```
Comment (DerAlexmeister) [inline iocextractor/core/server.py:278]: See line 280

**Review by DerAlexmeister — COMMENTED** [review]
Start the thread or schedular when the blacklist was fetched.

File: iocextractor/core/server.py (hunk 1 above)
Comment (p2h5) [reply iocextractor/core/server.py:7]: Do you mean I should undo the removing of the debug import?

File: iocextractor/core/server.py (hunk 2 above)
Comment (p2h5) [reply iocextractor/core/server.py]: Removed the duplicate.

File: iocextractor/core/server.py (hunk 1 above)
Comment (DerAlexmeister) [reply iocextractor/core/server.py:7]: Yes remove the import statement

File: iocextractor/core/server.py (hunk 2 above)
Comment (DerAlexmeister) [reply iocextractor/core/server.py]: Check for an not empty blacklist before starting the thread.

File: iocextractor/core/server.py (hunk 1 above)
Comment (p2h5) [reply iocextractor/core/server.py:7]: I thought I already did by the first commit.

File: iocextractor/core/server.py (hunk 2 above)
Comment (p2h5) [reply iocextractor/core/server.py]: I am doing so inside the refetch_blacklist on line 113. But I could additionally build this logic in the __call__ function if you want.

File: iocextractor/core/server.py (hunk 1 above)
Comment (DerAlexmeister) [reply iocextractor/core/server.py:7]: Yes fine.

File: iocextractor/core/server.py (hunk 2 above)
Comment (DerAlexmeister) [reply iocextractor/core/server.py]: Does this do the job?

Comment (p2h5) [reply iocextractor/core/server.py]: Should be working. Before starting the thread or the scheduler, we are stepping into the refetch_blacklist / refetch_datasources. If the Blacklist/ the Datasources are empty, we are getting the data locally. Else we are using gitlab to get the data.

**Pull request merged**